.gitignore

```diff
@@ -9,3 +9,4 @@
 /.mypy_cache
 /venv
 /recordings
+.vscode/settings.json
```

Comment:
I don't think this is needed? We already ignore `/.vscode` as a whole folder. Did you have a case where this wasn't being ignored?

Reply:
I did have a case where the settings file still wanted to be added, but this didn't fix it so there's not much point